engine/interpreter/src/test/scala/pl/touk/nussknacker/engine/compile/ProcessValidatorSpec.scala

```diff
@@ -93,11 +93,51 @@ class ProcessValidatorSpec extends FunSuite with Matchers with Inside {
     ObjectDefinition.noParam,
     ExpressionDefinition(
       Map("processHelper" -> ObjectDefinition(List(), Typed(ProcessHelper.getClass), List("cat1"), SingleNodeConfig.zero)),
-      List.empty, LanguageConfiguration.default, optimizeCompilation = false, strictTypeChecking = true, dictionaries = Map.empty, hideMetaVariable = false, strictMethodsChecking = true
+      List.empty, LanguageConfiguration.default, optimizeCompilation = false, strictTypeChecking = true, dictionaries = Map.empty, hideMetaVariable = false, strictMethodsChecking = true, staticMethodInvocationsChecking = true
     ),
     ClassExtractionSettings.Default
   )
 
+  test("Invocation of Type Reference of valid class, should succeed") {
+
+    val typeReferenceWithValidClass = "T(MathContext).staticMethod()"
```

Comment:
1. What does make `MathContext` available for typing? It is available because of `Decimal`? Can we use some more straightforward type like `String` + write comment that `String` is always available or sth like that?
2. This test is tests invalid case - AFIK spel need full class name (with package). Even if this expression will validate, it will thow exception during evaluation. I think that we need to add more tests in `SpelExpressionSpec` that would check that not only epxression is valid but also would evaluate

Reply:
changed to String